client/src/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App.jsx';
+
+ReactDOM.render(<App />, document.getElementById('app'));
```

Comment:
element id should be 'menu' or something similar, no?

Reply:
I actually was debating whether to put App or RestaurantMenu or similar but ended up just putting a generic 'App' title. I've removed the file for now and will work on this on a separate branch.